web/src/components/dashboard/DashboardOnDesktop.tsx

```diff
@@ -81,7 +81,9 @@ export const DashboardOnDesktop = (props: Props): ReactElement => {
                   <>
                     <hr className="margin-bottom-3" />
                     <Heading level={2}>
-                      {Config.dashboardRoadmapHeaderDefaults.RoadmapTasksHeaderText}
+                      {operatingPhase.id === OperatingPhaseId.DOMESTIC_EMPLOYER
```

Comment:
pebble - So this code is repeated verbatim in the DashboardOnMobile code. I think as a result it would be good to make it a function. 

Also its business logic at this point and has nothing to do with the actual shape of the component and should thus, be separated into a function somewhere else anyways